portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,11 +32,20 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> comments = new ArrayList<String>();    
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
```

Comment:
Would consider extracting "Comment" and "text" out as constants since they're used in multiple places as such

Reply:
Thank you!